src/js/media-time-display.js

```diff
@@ -64,8 +67,12 @@ class MediaTimeDisplay extends MediaTextDisplay {
 
   constructor() {
     super();
-    this.container.style.cursor = 'pointer';
-    this.container.innerHTML = `<span style="cursor: pointer">${formatTimesLabel(this)}</span>`;
+
+    this.#slot = this.shadowRoot.querySelector('slot');
+    this.#slot.innerHTML = `${formatTimesLabel(this)}`;
```

Comment:
suggestion(non-blocking):

```suggestion
this.#slot.textContent = `${formatTimesLabel(this)}`;
```